src/StudyEntities/Candidate/ValidatableIdentifier.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace LORIS\StudyEntities\Candidate;
+
+abstract class ValidatableIdentifier implements Identifier
+{
+    protected $value;
+
+    abstract static protected function validate(string $value): bool;
```

Comment:
comment here about what this is expected of validate, and maybe an explanation for the reasons of it being 'static validate(string)` and not `validate()`